DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/SetLearningLogItemCompletionDateViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+{
+    public class SetLearningLogItemCompletionDateViewModel : SetLearningLogItemCompletionDateFormData
+    {
+        public SetLearningLogItemCompletionDateViewModel(
+            int learningLogItemId,
+            string resourceName
+        )
+        {
+            LearningLogItemId = learningLogItemId;
+            ResourceName = resourceName;
+        }
+
+        public SetLearningLogItemCompletionDateViewModel(
+            SetLearningLogItemCompletionDateFormData formData,
+            int learningLogItemId
+        ) : base(formData)
+        {
+            LearningLogItemId = learningLogItemId;
+        }
+
+        public int LearningLogItemId { get; set; }

```

Comment:
Looks like there is a hidden input for the LearningLogItemId, so I'd think it would be returned in the post, and thus should be part of the FormData. But then everything useful would be in form data and this class would be a somewhat pointless wrapper. Are we good to include everything in one class if that hidden input stays?

Reply:
Ah whoops, I didn't mean to have the hidden for it. I'll just remove that